Foundation/NSFileManager.swift

```diff
@@ -712,9 +712,15 @@ extension FileManager {
 }
 
 extension FileManager {
-    open var homeDirectoryForCurrentUser: URL { NSUnimplemented() }
+    open var homeDirectoryForCurrentUser: URL {
+        return homeDirectory(forUser: CFCopyUserName().takeRetainedValue()._swiftObject)!
```

Comment:
Hm, is a force unwrap the right answer here if we can't find a home directory? Maybe a fallback instead?